js/client-battle.js

```diff
@@ -571,13 +571,14 @@
 				if (this.choice && this.choice.waiting && !this.finalDecision) {
 					buf += '<button name="undoChoice">Cancel</button>';
 				}
-				buf += '</p>';
+				buf += '</p><p class="timer">';
```

Comment:
May I suggest using using a tone of gray instead of green, for the options color?
Might blend better with the UI colors imo